frontend-react/src/pages/admin/InternalUserGuides.tsx

```diff
@@ -12,9 +12,7 @@ export const InternalUserGuidesDirectory = [
 ];
 
 const InternalUserGuides = () => {
-    return (
-        <StaticPagesFromDirectories directories={InternalUserGuidesDirectory} />
-    );
+    return <IASideNavTemplate directories={InternalUserGuidesDirectory} />;
```

Comment:
are we not using an IATemplate here because we don't need the header? Is there a common pattern to go directly to a specific template for a page (like in this case), or should we expect everything to be wrapped in an IATemplate?

Reply:
This is actually a completely unused page. Removing it.